packages/connectors/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@fuels/connectors",
+  "version": "0.0.1",
+  "main": "./src/index.ts",
+  "publishConfig": {
+    "main": "./dist/index.js",
+    "module": "./dist/index.mjs",
+    "types": "./dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "tsup --dts",
+    "ts:check": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@fuel-connectors/fuel-development-wallet": "workspace:*",
+    "@fuel-connectors/fuel-wallet": "workspace:*",
+    "@fuel-connectors/fuelet-wallet": "workspace:*"
+  },
```

Comment:
I think we need to move the dependencies here to devDependencies in order for tsup to create a bundle without this dependencies. You can test how this is going to look like by running `pnpm pack` inside this folder after running `pnpm build`.

Reply:
https://github.com/FuelLabs/fuel-connectors/pull/1/commits/f37b1d4455f9ae985b95abdf37685acba98ef19b